### ISSUE TITLE
paper-toast accessibility issues

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-toast#4]
Keyboard navigation issues:

-  I am unable to use just the keyboard to navigate through the buttons and activate one to trigger the toast. When I am using a screen reader, I am able to access the buttons and use the force click key command to trigger the toast, but enter does not work.

Focus:
- The animation on the buttons only appears when I use the mouse to click, not when I use the keyboard.
 
Toast Issues:
- The toasts are not being read aloud by screen readers. These need to be read as announcements
 
- In the Get Messages toast, there is the ability to click "retry". This is not keyboard accessible. If there is an action available within the toast, we need to ensure a user can get to it with the keyboard. 

- The toast times out. If a screen reader is enabled, we should consider extending the length of time it is displayed.
 
- I am unable to trigger the toast at all on Chrome OS, even when clicking the button with a mouse.
 

Windows Specific issues with NVDA:
 
- Chrome / NVDA: I can access the buttons but NVDA reads them as one: "button discard draft button get messages". This is a confusing user experience.
 
- Firefox / NVDA: there is a "blank" announced between the two buttons.

**Comment by lpalmaro** [comment]
Just tested again. With Voiceover/Chrome: I'm still not able to access the toasts with the screen reader

On Chrome OS / chromevox, I am able to access the toast messages, however, I have to navigate to them using shift + search + down arrow. If I didn't visually see that they were there, I wouldn't know to navigate to them. Could we make this an alert that speaks aloud? 

Still need to retest on Windows.

**Comment by lpalmaro** [comment]
FYI - just testing on Windows with JAWS and Firefox. This seems to work pretty well -- when I activate the buttons, I do hear the toast verbalized. I can also press tab to reach the retry button within the toast. However, the length of the toast is pretty short, especially for someone who is using a screen reader or someone who is low vision. Any specific reason for the current length of the toast appearing? Could we explore making it stay on screen for a few extra seconds?

**Comment by notwaldorf** [comment]
@valdrinkoshi Would you mind checking if these are still an issue, or if they have been fixed? Thanks!

**Comment by valdrinkoshi** [comment]
All seems to be fixed except the duration length.
We can either update the demo page to have toasts with longer duration (non-breaking change) or change the default value (breaking change).
